web/pdf_viewer.css

```diff
@@ -49,6 +48,7 @@
 
 .pdfViewer {
   padding-bottom: var(--pdfViewer-padding-bottom);
+  height: 100%;
```

Comment:
I really don't understand how this is correct, since the pages now overflow the `.pdfViewer` element which really doesn't seem right since they should be *contained* within the actual viewer.
Sorry, but this doesn't seem like a generally correct/safe change here!